src/cml.js

```diff
@@ -299,12 +298,30 @@ class CML {
         } else if (duration) {
           log.status = 'job_ended';
           log.success = msg.includes('Job succeeded');
-          log.level = log.success ? 'info' : 'error';
```

Comment:
Same as above

Reply:
I have added the line later on to just have one setup